api-2.0/src/main/java/org/openmrs/module/attachments/obs/DefaultAttachmentHandler2_0.java

```diff
@@ -0,0 +1,14 @@
+package org.openmrs.module.attachments.obs;
+
+import org.openmrs.obs.handler.BinaryDataHandler;
+
+public class DefaultAttachmentHandler2_0 extends DefaultAttachmentHandler {
+	
+	public DefaultAttachmentHandler2_0() {
+		super();
+	}
+	
+	protected void setParentComplexObsHandler() {
+		setParent(new BinaryDataHandler());
+	}
+}
```

Comment:
Why do you need this?

Reply:
Due to the error I was facing I had thought the implementing class BinaryHandler hadnt implemented...Let me push again though with a similar error.